app/bundles/CoreBundle/Resources/views/Modal/token_placeholders.html.twig

```diff
@@ -0,0 +1,611 @@
+<!-- Token placeholders modal -->
```

Comment:
How about breaking it into smaller and dedicated files for 
- Contact fields
- Company Contact fields
- Mautic Component tokens
- Email specific tokens
- Landing Page tokens
- Preference Center Landing Page tokens
- Dynamic Web Content tokens
- Contact Monitoring
- Custom fields
This will help with the readability and maintainability.

Reply:
@escopecz  Where (bundle) do I place the component tokens file, as we have several different components here?

![image](https://github.com/user-attachments/assets/22cec5bf-35e4-4b99-8dd8-308e6ad170dc)

Reply:
Tokens are tricky. Because some tokens work in the email but not in pages and vice versa. So I'd generate this help for only the places where tokens can be used and show only the tokens available. 

And again, this help text can be auto-generated. There are already ajax calls every time you open a page edit form, email edit form, SMS edit form, ... that are loading available tokens for the builders and textareas. We can generate the help texts from that same information.

Reply:
Hm, the issue is that we'd fall again into the problem of information disappearing as we currently have with search commands... when things are tricky, we have a circus and the user is the clown

The ideal scenario here would be to specify: I can add a small description to tell users where they can use certain groups of tokens 

_which is also not exact information, see #14099, 
or: I recently discovered that we can use other tokens like /thank-you?email={formfield=email} in the redirect URL to help identify the user or trigger other functionalities_